guides/plugins/apps/content/cms/add-custom-cms-blocks.md

```diff
@@ -20,36 +20,38 @@ The basic directory structure looks as follows:
 
 ```text
 ├── Resources
+│   ├── app
+│   │   └── storefront
+│   │       └── src
+│   │           └── scss
+│   │               └── base.scss
 │   ├── cms
 │   │   └── blocks
-│   │       ├── my-first-block
+│   │       ├── swag-image-text-reversed
 │   │       │   ├── preview.html
 │   │       │   └── styles.css
-│   │       └── my-second-block
-│   │           ├── preview.html
-│   │           └── styles.css
 │   ├── views
 │   │   └── storefront
-│   │       └── cms
-│   │           └── block
-│   │               ├── my-first-block.html.twig
-│   │               └── my-second-block.html.twig
+│   │       └── block
+│   │           └── cms-block-swag-image-text-reversed-component.html.twig
 │   └── cms.xml
 └── manifest.xml
 ```
 
 Each CMS block defined within your `cms.xml` must have a directory matching the block's name in `Resources/cms/blocks/`.
-In those directories you shape your blocks by supplying a `preview.html` containing the template used for displaying a preview.
-Styling the preview is possible from the `styles.css`.
+In those directories you shape your blocks for the CMS editor in the Administration by supplying a `preview.html` containing the template used for displaying a preview.
+Styling the preview in the sidebar and the component on the CMS stage is possible from the `styles.css`.
```

Comment:
`CMS stage` sound a bit weird to me, what about `CMS page`?

Reply:
The CMS stage is the part where you drag-and-drop your blocks into.

But you're kind of right it's not used anywhere else in the docs and might be somewhat confusing.

Reply:
I'll rename it to `CMS editor` instead of `CMS stage` - I think this is more comprehensible.

Reply:
CMS panel? 🤔